src/components/clusterConfiguration/DiscoveryTroubleshootingModal.tsx

```diff
@@ -50,78 +50,66 @@ export const DiscoveryTroubleshootingModal: React.FC<DiscoveryTroubleshootingMod
-        />
-        <Text component={TextVariants.p}>If you see something like this:</Text>
-        <PrismCode
-          code={`May 17 14:01:14 localhost systemd[1]: Starting agent.service...
-May 17 14:01:18 localhost docker[1013]: Unable to find image locally
-May 17 14:01:33 localhost docker[1013]: /usr/bin/docker: Error response from daemon: Get https://quay.io/v2/: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers).
-May 17 14:01:33 localhost docker[1013]: See '/usr/bin/docker run --help'.
-May 17 14:01:33 localhost systemd[1]: agent.service: Control process exited, code=exited, status=125/n/a
-May 17 14:01:33 localhost systemd[1]: agent.service: Failed with result 'exit-code'.
-May 17 14:01:33 localhost systemd[1]: Failed to start agent.service.`}
+          code={`Oct 15 11:26:35 localhost systemd[1]: agent.service: Service RestartSec=3s expired, scheduling restart.
+Oct 15 11:26:35 localhost systemd[1]: agent.service: Scheduled restart job, restart counter is at 9.
+Oct 15 11:26:35 localhost systemd[1]: Stopped agent.service.
+Oct 15 11:26:35 localhost systemd[1]: Starting agent.service...
+Oct 15 11:26:35 localhost podman[1834]: Trying to pull quay.io/ocpmetal/assisted-installer-agent:latest...
+Oct 15 11:26:35 localhost podman[1834]:   Get "https://quay.io/v2/": dial tcp: lookup quay.io on [::1]:53: read udp [::1]:58297->[::1]:53: read: connection refused
+Oct 15 11:26:35 localhost podman[1834]: Error: unable to pull quay.io/ocpmetal/assisted-installer-agent:latest: unable to pull image: Error initializing source docker://quay.io/ocpmetal/assisted-installer-agent:latest: error pinging dock>
+Oct 15 11:26:35 localhost systemd[1]: agent.service: Control process exited, code=exited status=125
+Oct 15 11:26:35 localhost systemd[1]: agent.service: Failed with result 'exit-code'.
+Oct 15 11:26:35 localhost systemd[1]: Failed to start agent.service.`}
         />
         <Text component={TextVariants.p}>
-          That implies you have a networking issue, perhaps a wrong proxy or no connection to the
-          assisted installation service.You can use the nmcli to get additional information about
-          your network configuration.
+          Check the proxy settings and verify that the assisted installer service is connected to a
+          network. You can use <code>nmcli</code> to get additional information about your network
+          configuration.
         </Text>
-        <Text component={TextVariants.h2}>Check agent logs</Text>
-        <PrismCode code={`[core@vm-11-47 ~]$ less /var/log/agent.log`} />
         <Text component={TextVariants.h2}>Check assisted-installer logs</Text>
         <PrismCode
-          code={`[core@vm-11-47 ~]$ sudo su
-[core@vm-11-47 ~]$ podman ps -a | grep assisted-installer
-[core@vm-11-47 ~]$ podman logs <container id>`}
+          code={`sudo su
+podman ps -a | grep assisted-installer
```

Comment:
just wondering if we should be consistent (above we use ps -ef :)

Reply:
Hmm, it is `ps -ef` vs `podman ps -a` which are different commands.